src/components/task/workflows/WorkflowForm.js

```diff
@@ -80,13 +82,15 @@ const fields = ({ itemId, workflowType }) => ({ workflow }) =>
       required: false,
       options: ({ sources }) => sources,
       initialValue: workflow ? workflow.get('sourceName') : '',
+      helpText: 'The application that is calling and getting the results back from the workflow. The source provides information that can be used in parameters and connectors to pass information or do logic.',
     },
     {
       name: 'sourceGroup',
       label: 'Source Group',
       type: 'text',
       required: false,
       initialValue: workflow ? workflow.get('sourceGroup') : '',
+      helpText: 'Categorization within the Source. For Request CE it\'s the combination of the type (submission of form), Kapp Slug and the Form Slug separated by a greater than sign ( > ). Example: Submissions > services > onboarding.',
```

Comment:
Slight change of the first sentence:

Categorization of the workflow based on rules provided by the Source.